posthog/api/capture.py

```diff
@@ -183,14 +183,16 @@ def get_event(request):
                 sent_at=sent_at,
             )
 
-        process_event.delay(
-            distinct_id=distinct_id,
-            ip=get_ip_address(request),
-            site_url=request.build_absolute_uri("/")[:-1],
-            data=event,
-            team_id=team.id,
-            now=now,
-            sent_at=sent_at,
-        )
+        # Selectively block certain teams from having events published to Postgres on Posthog Cloud
+        if not getattr(settings, "MULTI_TENANCY", False) or team.id not in [536, 572]:
```

Comment:
I think long-term this is worth storing this in redis (or pg) and polling it every few minutes inside the worker.

Reply:
@fuziontech Actually the way we talked about doing this earlier was to block certain signup_tokens. This way if anyone is self-hosting and they're getting spam on a signup token they can block those in a ENV var to avoid DB lookups etc.